<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPatientDialog</class>
 <widget class="QDialog" name="AddPatientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Patient</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="healthCareNumberLabel">
       <property name="text">
        <string>Health Care Number</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="healthCareNumberLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="typeOfCareLabel">
         <property name="text">
          <string>Type of care</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="acuteCareRadioButton">
         <property name="text">
          <string>AC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="complexContinuingCareRadioButton">
         <property name="text">
          <string>CCC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="longTermCareRadioButton">
         <property name="text">
          <string>LTC</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="waitingListsLabel">
       <property name="text">
        <string>Waiting Lists</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QListWidget" name="waitingListListWidget">
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
       <item>
        <property name="text">
         <string>Renfrew County</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>North Lanark/North Grenville</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ottawa West</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ottawa Center</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ottawa East</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Eastern Counties</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddPatientDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddPatientDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
